src/hooks/custom-hooks/useTrackjs.ts

```diff
@@ -3,17 +3,23 @@ import { useAuthData } from '@deriv-com/api-hooks';
 
 const { VITE_TRACKJS_TOKEN } = process.env;
 
+/**
+ * Custom hook to initialize TrackJS.
+ * @returns {Object} An object containing the `init` function.
+ */
 const useTrackjs = () => {
     const { activeLoginid } = useAuthData();
     const init = () => {
-        TrackJS.install({
-            application: 'deriv-p2p',
-            dedupe: false,
-            enabled: location.host.indexOf('localhost') !== 0,
-            token: VITE_TRACKJS_TOKEN!,
-            userId: activeLoginid || 'undefined',
-            version: (document.querySelector('meta[name=version]') as HTMLMetaElement)?.content,
-        });
+        if (!TrackJS.isInstalled()) {
+            TrackJS.install({
+                application: 'deriv-p2p',
+                dedupe: false,
+                enabled: location.hostname !== 'localhost',
+                token: VITE_TRACKJS_TOKEN!,
+                userId: activeLoginid || 'undefined',
+                version: (document.querySelector('meta[name=version]') as HTMLMetaElement)?.content,
```

Comment:
[Q] what happen if meta is not available?, can we put it as 'undefined' as well here?